website/static/css/commentpane.css

```diff
@@ -122,3 +122,20 @@
 .comment-gravatar {
     height: 20px;
 }
+
+.atwho-input {
+    display: block;
+    height: auto;
+    min-height: 56px;
+    overflow: hidden;
+    word-wrap: break-word;
+}
+
+.atwho-input[placeholder]:empty:before {
+    content: attr(placeholder);
+    color: #555;
```

Comment:
Minor: The placeholder text should be a more muted color than input text. 

In this PR:

![osf___small_project](https://cloud.githubusercontent.com/assets/2379650/16427565/dd6557d2-3d3b-11e6-94ae-2e1601cbdaf6.png)


On prod (Chrome's native placeholder styling):

![osf___small_project](https://cloud.githubusercontent.com/assets/2379650/16427543/ca3d7892-3d3b-11e6-805d-0a2fd716ea8c.png)